src/components/survey.js

```diff
@@ -14,12 +14,14 @@ module.exports = function(app) {
           angular.forEach(component.values, function(v) {
             values[v.value] = v.label;
           });
-          angular.forEach(component.questions, function(question) {
-            view += '<tr>';
-            view += '<th>' + question.label + '</th>';
-            view += '<td>' + values[data[question.value]] + '</td>';
-            view += '</tr>';
-          });
+		  if(data) {
```

Comment:
I am also not sure this will fix the problem. The issue is that the ```values[data[question.value]]``` is not printing the actual values for these fields, but rather the labels of the questions. Please see the video posted by jeriah.